### PR TITLE
adding block

### PR DESCRIPTION
**Pull request by iruzevic**
### Added
- New video-iframe component.
- New Vimeo block.
- New Youtube block.

Screenshot 2020-07-09 at 13 55 41
Screenshot 2020-07-09 at 13 55 56
Screenshot 2020-07-09 at 13 56 04

### CODE REVIEW

File: blocks/init/src/blocks/components/video-iframe/components/video-iframe-editor.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  return (
+    <div className={videoClass}>
+      <iframe
+        className={`${componentClass}__iframe`}
+        src={`${url}${id}`}
+        title={title}
+        frameBorder="0"
+        allow="autoplay; fullscreen"
+        allowFullScreen
```
Comment (dingo-d) [inline blocks/init/src/blocks/components/video-iframe/components/video-iframe-editor.js:29]: Should these be properties as well?

Comment (IvanGrginovInf) [reply blocks/init/src/blocks/components/video-iframe/components/video-iframe-editor.js:29]: This is just in the editor I believe, on frontend you be oembed's output.

Comment (dingo-d) [reply blocks/init/src/blocks/components/video-iframe/components/video-iframe-editor.js:29]: I was thinking about the editor, like if you'd like to change the autoplay option etc.

File: blocks/init/src/blocks/custom/youtube/docs/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Youtube Block
+
+Block used to provide quote functionality to your project.
```
Comment (gabriel-glo) [inline blocks/init/src/blocks/custom/youtube/docs/readme.md:3]:
```suggestion
Block used to provide YouTube video embed functionality to your project.
```

File: blocks/init/src/blocks/custom/youtube/docs/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Youtube Block
```
Comment (gabriel-glo) [inline blocks/init/src/blocks/custom/youtube/docs/readme.md:1]:
```suggestion
# YouTube Block
```

File: blocks/init/src/blocks/custom/youtube/youtube.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Template for the Youtube Block view.
```
Comment (gabriel-glo) [inline blocks/init/src/blocks/custom/youtube/youtube.php:3]:
```suggestion
 * Template for the YouTube Block view.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,11 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 ## [Unreleased]
 
+### Added
+- New video-iframe component.
+- New Vimeo block.
+- New Youtube block.
```
Comment (gabriel-glo) [inline CHANGELOG.md:12]:
```suggestion
- New YouTube block.
```

**Pull request merged**